timeboost/auctioneer.go

```diff
@@ -426,6 +443,25 @@ func (a *AuctioneerServer) resolveAuction(ctx context.Context) error {
 	return nil
 }
 
+func (a *AuctioneerServer) acknowledgeAllBids(ctx context.Context, round uint64) {
+	a.unackedBidsMutex.Lock()
+	defer a.unackedBidsMutex.Unlock()
+
+	var acknowledgedCount int
+	for msgID, bid := range a.unackedBids {
+		if uint64(bid.Round) <= round {
+			if err := a.consumer.SetResult(ctx, msgID, nil); err != nil {
+				log.Error("Error acknowledging bid after auction resolution", "msgID", msgID, "error", err)
+				continue
```

Comment:
if we continue because of an error here, is it ok if we have an unacked bid that is not cleared in the map?